docker.go

```diff
@@ -133,7 +137,12 @@ func (p Plugin) Exec() error {
 		cmds = append(cmds, commandPull(img))
 	}
 
-	cmds = append(cmds, commandBuild(p.Build)) // docker build
+    if p.Loadimage == false {
```

Comment:
could it be extracted to the named function/lambda? It would better explain what is happening int this piece of code.

Reply:
hmmm... I'd personally rather leave it as is, but there is definitely some point in getting the notion of producing image out there. Anyone has strong opinions on this?